mmv1/products/networkservices/TcpRoute.yaml

```diff
@@ -0,0 +1,157 @@
+    path: 'error'
+    message: 'message'
+autogen_async: true
+import_format: ['projects/{{project}}/locations/global/tcpRoutes/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_tcp_route_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-tcp-route'
+      backend_service_name: "my-backend-service"
+      http_health_check_name: "backend-service-health-check"
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_tcp_route_actions'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-tcp-route'
+      backend_service_name: "my-backend-service"
+      http_health_check_name: "backend-service-health-check"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    url_param_only: true
+    immutable: true
+    description: |
+      Name of the TcpRoute resource.
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'selfLink'
+    description: |
+      Server-defined URL of this resource.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    description: |
+      Time the TcpRoute was created in UTC.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    description: |
+      Time the TcpRoute was updated in UTC.
+    output: true
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: 'labels'
+    description: Set of label tags associated with the TcpRoute resource.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      A free-text description of the resource. Max length 1024 characters.
+  - !ruby/object:Api::Type::Array
+    name: meshes
+    item_type: Api::Type::String
+    description: |
+          List of meshes this TcpRoute is attached to, as one of the routing rules to route the requests served by the mesh.  
+    send_empty_value: true 
+  - !ruby/object:Api::Type::Array
+    name: gateways
+    item_type: Api::Type::String
+    description: |
+          List of gateways this TcpRoute is attached to, as one of the routing rules to route the requests served by the gateway.  
+    send_empty_value: true
+  - !ruby/object:Api::Type::Array
+    name: rules
+    description: |
+          Rules that define how traffic is routed and handled.  
+    required: true
+    send_empty_value: true
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties: 
+        - !ruby/object:Api::Type::Array
+          name: matches
+          description: |
+            Matches define conditions used for matching the rule against incoming TCP requests.
+          send_empty_value: true
+          item_type: !ruby/object:Api::Type::NestedObject
+            properties:
+              - !ruby/object:Api::Type::String
+                name: address
+                description: |
+                  Required. Must be specified in the CIDR range format.
```

Comment:
`Requird` is not needed in `description`. It will show required in the user documentation because of the field `required: true` 

https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-7656-old..auto-pr-7656#diff-55b40c86473766ed36c5d877aba2830aff1b9b983044629b202bec405b93b710R149

Reply:
Removed